sys/select_bsd.go

```diff
@@ -4,6 +4,13 @@ package sys
 
 import "syscall"
 
+const NFDBits = 32
```

Comment:
unsafe.Sizeof(u) returns 8 on both Linux and MacOS. But NFDBits needs to be 64 on Linux and 32 on MacOS. Since we have to separate the impl anyway I just left NFDBits hard-coded.